controllers/homeController.js

```diff
@@ -76,6 +87,31 @@ var findCommonChampions = function(championsA, championsB) {
     return commonChampions
 }
 
+var compareCommonChampions = function(commonChampions) {
+    var aCount = 0;
+    var bCount = 0;
+    commonChampions.forEach(function(championTuple) {
```

Comment:
Sounds good. How about something like this as the returning JSON:
```
result = {
    compare : // postive for champion A winning, negative for champion B winning, 0 for tie.
    reason: {
        winCompare:  // value or values for why they won in this "category". E.g. in this case, how many more wins one had than the other.

        // other comparison categories
    }
}
```

This allows us to know the winner immediately by looking at the compare value, and also know the reasons in the future.